src/devices/sound/k051649.cpp

```diff
@@ -115,7 +149,7 @@ void k051649_device::sound_stream_update(sound_stream &stream, stream_sample_t *
 			int c=voice.counter;
 			int step = ((int64_t(m_mclock) << FREQ_BITS) / float((voice.frequency + 1) * 16 * (m_rate / 32))) + 0.5f;
 
-			short *mix = m_mixer_buffer.get();
+			short *mix = m_mixer_buffer.begin();
```

Comment:
Are you sure this compiles?  I would've thought you'd either need to change the type of `mix` from `short *` to `std::vector<short>::iterator` or use `&m_mixer_buffer[0]` to initialise it.